Deprecate threads-video-downloader extension

ridemountainpig opened the pull request:
## Description
The Threads Video Downloader extension will be combined with the Threads extension in #15875. This pull request is created to deprecate the Threads Video Downloader extension.
<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

## Screencast

<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

LitoMore commented:
You can clean up the code, give users a `confirmAlert`, and launch the new extension.

ridemountainpig commented:
> You can clean up the code, give users a `confirmAlert`, and launch the new extension.

@LitoMore I have cleaned up the code and added the `crossLaunchCommand` action for the Threads extension. Can you please check it? Thank you.

LitoMore commented:
Cross-launch is unneeded here. You can simply open the deeplink `raycast://extensions/chrismessina/threads` to lead user to the extension installation page.

For users who have already installed Threads, their page will show the Installed message.

pernielsentikaer reviewed (approved):
Hi 👋 

Looks good to me, approved 🔥
